src/easyLess.ts

```diff
@@ -11,6 +12,8 @@ let lessDiagnosticCollection: vscode.DiagnosticCollection;
 export function activate(context: vscode.ExtensionContext) {
   lessDiagnosticCollection = vscode.languages.createDiagnosticCollection();
 
+  let preprocessor: Configuration.Preprocessor | undefined = undefined;
```

Comment:
I would like to allow for more than one pre-processor. Can this please be an array that gets pushed into by `registerPreprocessor()`?